test/evergreen.yml

```diff
@@ -5323,6 +5323,40 @@ buildvariants:
     - name: unit-test
     - name: fops
 
+- name: macos-1100-perf-tests
+  display_name: "OS X 11.00 Performance Tests"
+  run_on:
+  - macos-1100
+  batchtime: 120 # 2 hours
+  expansions:
+    CMAKE_INSTALL_PREFIX: -DCMAKE_INSTALL_PREFIX=$(pwd)/cmake_build/LOCAL_INSTALL
+    python_binary: '/opt/mongodbtoolchain/v3/bin/python3'
+    pip3_binary: '/opt/mongodbtoolchain/v3/bin/pip3'
+    virtualenv_binary: '/opt/mongodbtoolchain/v3/bin/virtualenv'
+    smp_command: -j $(sysctl -n hw.logicalcpu)
+    cmake_generator: "Unix Makefiles"
+    make_command: make
+    test_env_vars:
+      WT_BUILDDIR=$(git rev-parse --show-toplevel)/cmake_build
+      DYLD_LIBRARY_PATH=$WT_BUILDDIR
+    # Must disable TCMALLOC as it may be picked up locally and its not on all hosts.
```

Comment:
np: This comment should be two lines lower just before `posix_configure_flags: -DENABLE_TCMALLOC=0`

Reply:
Done! [da10f82](https://github.com/wiredtiger/wiredtiger/pull/8525/commits/da10f825757b66dd1477adf3df976f3211ececa3)